### DISCUSSION TITLE
How to integrate completions with fish shell?

### DISCUSSION THREAD

**Discussion by munntjlx** [JanDeDobbeleer/oh-my-posh#1960] [Troubleshoot]
I did figure out how to generate the completions file, but when I put on ~/.config/fish/completions/oh-my-posh nothing happens. Any way to write a quick faq on this? I can write if someone tells me how to do it.

**Comment by JanDeDobbeleer** [comment]
Good question as I wasn't able to find that documentation looking at Cobra's docs 😆 

Untested, based on other tools: To do so in all your shell sessions, add the following line to your ~/.config/fish/config.fish file:

```
oh-my-posh completion fish | source
```

**Reply by abbaty48** [reply]
helpful
